src/App.vue

```diff
@@ -41,6 +41,7 @@ html, body {
 h1,h2,h3 {
   font-family: 'Raleway', 'Regular', Helvetica, Arial, sans-serif;
```

Comment:
We aren't loading in the 'Raleway' font anywhere.

We should add into the head `<link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">` if we are going to use it

Reply:
fixed